common/config/src/raw.rs

```diff
@@ -303,6 +308,10 @@ pub struct LogConfig {
     pub log_metric_server_stats: Option<String>,
     #[serde(skip_serializing_if = "Option::is_none")]
     pub clear_cache_interval: Option<u32>,
+    #[serde(skip_serializing_if = "Option::is_none")]
```

Comment:
The way this raw config comes in again I think id would be great to have a little documentation around this.